Directory.Packages.props

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
```

Comment:
Do you want to enable that proactively? Just so you know it probably does not do anything right now as it only works for `<PackageReference>`. So right now we don't actually need this change

Reply:
@a-ctor am confused: the problem exists in Remotion.Web.Development.WebTesting via transitive dependency and this project is using SDK Style project format with PackageReference.